java/client/src/main/java/glide/api/GlideClusterClient.java

```diff
@@ -1292,36 +1322,75 @@ public CompletableFuture<GlideString> randomKeyBinary() {
                 RandomKey, new GlideString[0], this::handleGlideStringOrNullResponse);
```

Comment:
We should have a constant to represent an empty GlideString array to reduce allocations.